src/plugins/twitter/_base.scss

```
@@ -3,33 +3,78 @@
  * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
  */
 
-.wb-disable {
-	.wb-twitter {
-		.twitter-timeline {
-			text-indent: 0 !important;
+.wb-twitter {
+	/* Align the loading icon relative to the center of the Twitter link in browsers that support :has() */
+	&:has( a.twitter-timeline + .twitter-timeline-loading ) {
+		display: inline-block;
+
+		a {
+			&.twitter-timeline {
+				+ {
+					.twitter-timeline-loading {
+						&::before {
+							left: 0;
+							right: 0;
+						}
+
+						&::after {
+							margin: auto;
+						}
+					}
+				}
+			}
 		}
 	}
-}
 
-.wb-twitter {
-	.twitter-timeline {
-		display: block;
-	}
+	/* Display the loading icon as normal content */
+	a {
+		&.twitter-timeline {
+			+ {
+				.twitter-timeline-loading {
+					@extend %global-loading;
+					margin-top: 5px;
+					min: {
+						height: 100px;
+						width: 100px;
+					}
+					position: relative;
+
+					/* Align the loading icon leftwards by default */
+					&::before {
+						right: auto;
+					}
 
-	iframe {
-		&[style] {
-			width: 100% !important;
+					&::after {
+						margin-left: calc( ( 100px - 1em ) / 2 );
```

Stylelint error: Unexpected whitespace after "(" in a single-line function

```suggestion
						margin-left: calc(( 100px - 1em ) / 2 );
```

---

I came super close to doing that locally, but noticed that the trailing white space after ``2`` wasn't triggering an error (even though [``function-parentheses-space-inside``'s docs](https://stylelint.io/user-guide/rules/function-parentheses-space-inside/) suggest that it's supposed to).

I'm gonna change it up to this:
```suggestion
						margin-left: calc((100px - 1em ) / 2);
```

That's in line with how all of WET and most of GCWeb's pre-existing ``calc()`` pseudo-classes are spaced-out.